src/Service.js

```diff
@@ -1,15 +1,15 @@
 const baseUrl = "https://feature-hunt-final.herokuapp.com/";
  //const baseUrl = "https://damp-citadel-25681.herokuapp.com/"
 
-//  const requestOptionsBuilder = (method, body, headers) => {
-//    let options = {
-//      method: method,
-//      headers: {'Content-Type': 'application/json'},
-//    };
-//    if (body) options['body'] = JSON.stringify(body);
-//    if (headers) options.headers = headers;
-//    return options;
-// };
+ const requestOptionsBuilder = (method, body, headers) => {
+   let options = {
+       method: method,
+       headers: {}
+   };
+   if (body) options['body'] = JSON.stringify(body);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8494981)
```suggestion
   if (body) {options['body'] = JSON.stringify(body);}
```